CHANGELOG.md

```diff
@@ -17,6 +17,14 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [6.31.0] - 2023-10-06
+### Added
+- Retrieve method for session, `client.iam.session.retrieve`
+- Added option for getting a session as a context objet with `client.iam.session()`
```

Comment:
I don't see how or when "normal" users would need this (the context manager). I think this just could be a part of our test helper functions.